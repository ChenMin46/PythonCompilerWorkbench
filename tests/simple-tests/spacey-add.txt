# with extra spaces ...
c  =  a + b      
# and a trailing line comment
