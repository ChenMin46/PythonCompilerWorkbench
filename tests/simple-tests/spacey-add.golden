# with extra spaces ...
c  =  a + b      
# and a trailing line comment
===
Module
  body: [
    Assign >> 'c  =  a + b'
      targets: [
        Name >> 'c'
          id: 'c'
          ctx: Store
      ]
      value: BinOp >> 'a + b'
        left: Name >> 'a'
          id: 'a'
          ctx: Load
        op: Add
        right: Name >> 'b'
          id: 'b'
          ctx: Load
  ]
--- JSON ---
{
  "contents": [
    "", 
    {
      "token_kind": "comment", 
      "value": "# with extra spaces ..."
    }, 
    "\n", 
    "", 
    {
      "contents": [
        {
          "contents": [
            {
              "contents": [
                "c"
              ], 
              "id": "id_0", 
              "name": "Name"
            }, 
            "  =  ", 
            {
              "contents": [
                {
                  "contents": [
                    "a"
                  ], 
                  "id": "id_0", 
                  "name": "Name"
                }, 
                " + ", 
                {
                  "contents": [
                    "b"
                  ], 
                  "id": "id_0", 
                  "name": "Name"
                }
              ], 
              "id": "id_0", 
              "name": "BinOp"
            }
          ], 
          "id": "id_0", 
          "name": "Assign"
        }
      ], 
      "id": "id_0", 
      "name": "Module"
    }, 
    "      ", 
    "\n", 
    "", 
    {
      "token_kind": "comment", 
      "value": "# and a trailing line comment"
    }, 
    "\n", 
    ""
  ], 
  "id": "id_ROOT", 
  "name": "ROOT_NODE"
}
